mne/forward/_field_interpolation.py

```diff
@@ -87,6 +105,84 @@ def _compute_mapping_matrix(fmd, info):
     return mapping_mat
 
 
+def _as_type_evoked(evoked, to_type='grad', mode='fast', copy=True, n_jobs=1):
```

Comment:
I would name it _as_meg_type_evoked to make clear that it's for meg channels only